clients/go/pkg/auditopt/config.go

```diff
@@ -46,40 +46,17 @@ import (
 
 const DefaultConfigFilePath = "/etc/lumberjack/config.yaml"
 
-// MustFromConfigFile specifies a config file to configure the
-// audit client. `path` is required, and if the config file is
-// missing, we return an error.
-func MustFromConfigFile(path string) audit.Option {
-	return mustFromConfigFile(path, envconfig.OsLookuper())
-}
-
-// mustFromConfigFile is like MustFromConfigFile, but exposes a custom lookuper
-// for testing.
-func mustFromConfigFile(path string, lookuper envconfig.Lookuper) audit.Option {
-	return func(c *audit.Client) error {
-		fc, err := os.ReadFile(path)
-		if err != nil {
-			return err
-		}
-		cfg, err := loadConfig(fc, lookuper)
-		if err != nil {
-			return err
-		}
-		return clientFromConfig(c, cfg)
-	}
-}
```

Comment:
Why remove?

Reply:
I can't find the usage of `MustFromConfigFile` in the codebase.